### PR TITLE
Remove some unusued code

### PR DESCRIPTION
**Pull request by rwwagner90**
Just doing some initial cleanup before I work on tackling pulling in more of the data for the builds from the json and automating things.

### CODE REVIEW

**Comment by locks** [comment]
@rwwagner90 I also did a thing https://github.com/ember-learn/builds/pull/78/files 😅. What do you think about landing mine and then rebasing yours on that work?

**Comment by rwwagner90** [comment]
@locks ah man, I checked to make sure there were no PRs before I started. I guess you beat me too it 😛 . We can do that.

File: package.json [hunk 1]
```diff
@@ -45,7 +46,7 @@
     "ember-maybe-import-regenerator-for-testing": "^1.0.0",
     "ember-moment": "^7.5.0",
     "ember-resolver": "^4.4.0",
-    "ember-source": "https://s3.amazonaws.com/builds.emberjs.com/canary/shas/6f92209039f4f9a2c08c5642a02a4e598f3702cd.tgz",
+    "ember-source": "~3.1.0-beta.1",
```
Comment (locks) [inline package.json]: I'm confused by this, shouldn't my PR have this change? 🤔

File: package.json [hunk 2]
```diff
@@ -37,6 +37,7 @@
     "ember-cli-mirage": "^0.4.0",
     "ember-cli-moment-shim": "^3.4.0",
     "ember-cli-qunit": "^4.1.1",
+    "ember-cli-sass": "^7.1.5",
```
Comment (locks) [inline package.json]: could we avoid sass? node-sass gives me twitches

File: package.json (hunk 1 above)
Comment (rwwagner90) [reply package.json]: @locks I think yours was still on canary.

**Comment by rwwagner90** [comment]
@locks in the spirit of getting things done and shipping code, I removed the sass, styles, and link-to active stuff that I changed, and just kept the remaining few places I removed some dead code.

**Comment by locks** [comment]
:shipit:

**Pull request merged**